packages/app/screens/feed/create_group_form/CreateGroupFormScreen2.js

```diff
@@ -0,0 +1,178 @@
+import PropTypes from "prop-types";
+import React, { useState, useContext } from "react";
+import {
+  View,
+  Text,
+  TextInput,
+  TouchableWithoutFeedback,
+  Keyboard,
+  Switch,
+} from "react-native";
+import Button from "@app/components/Button";
+import Dropdown from "@app/components/Dropdown";
+import TagInput from "@app/components/TagInput";
+import { GroupContext } from "@app/utils/GroupContext";
+import { ScrollView } from "react-native-gesture-handler";
+import { styles } from "./CreateGroupFormStyles";
+
+function CreateGroupFormScreen2({ navigation }) {
+  const categoryOptions = ["Sports", "Education", "Business", "Gaming"];
+  const [inputValue, setInputValue] = useState("");
+  const toggleSwitch = () => setIsInviteOnly((previousState) => !previousState);
+
+  const {
+    category,
+    setCategory,
+    tags,
+    setTags,
+    location,
+    setLocation,
+    website,
+    setWebsite,
+    isInviteOnly,
+    setIsInviteOnly,
+  } = useContext(GroupContext);
+
+  const [categoryError, setCategoryError] = useState("");
+  const [tagsError, setTagsError] = useState("");
+  const [websiteError, setWebsiteError] = useState("");
+  // const [locationError, setLocationError] = useState("");
+
+  function handleInputValidationScreen2() {
+    let isValid = true;
+    const websiteRegex =
+      /^(?!(https?:\/\/)([\da-z.-]+)\.([a-z.]{2,6})([/\w .-]*)*\/?$).+$/i;
+
+    if (category.trim() === ``) {
+      setCategoryError(`Please select a category`);
+      isValid = false;
+    } else {
+      setCategoryError(``);
+    }
+
+    if (tags.length < 3) {
+      setTagsError(`Please select at least 3 tags`);
+      isValid = false;
+    } else {
+      setTagsError(``);
+    }
```

Comment:
Tags aren't mandatory for guilds in the database. I don't think we'll require guilds to define tags when being created.